backend/src/main/java/com/google/sps/eventsender/EventSender.java

```diff
@@ -0,0 +1,206 @@
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.client.util.DateTime;
+import com.google.api.client.util.store.FileDataStoreFactory;
+import com.google.api.services.calendar.Calendar;
+import com.google.api.services.calendar.CalendarScopes;
+import com.google.api.services.calendar.model.*;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.security.GeneralSecurityException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/** Instance of the class responsible for sending and adding events to user calendars */
+public class EventSender {
+
+  private static final String APPLICATION_NAME = "Ad-Lib";
+  private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+  private static final String TOKENS_DIRECTORY_PATH = "tokens";
+
+  /**
+   * Global instance of the scopes required by this quickstart. If modifying these scopes, delete
+   * your previously saved tokens/ folder.
+   */
+  private static final List<String> SCOPES = Collections.singletonList(CalendarScopes.CALENDAR);
+
+  private static final String CREDENTIALS_FILE_PATH = "/credentials.json";
+
+  /**
+   * Creates an authorized Credential object.
+   *
+   * @param HTTP_TRANSPORT The network HTTP Transport.
+   * @return An authorized Credential object.
+   * @throws IOException If the credentials.json file cannot be found.
+   */
+  private static Credential getCredentials(final NetHttpTransport HTTP_TRANSPORT)
+      throws IOException {
+    // Load client secrets.
+    InputStream in = EventSender.class.getResourceAsStream(CREDENTIALS_FILE_PATH);
+    if (in == null) {
+      throw new FileNotFoundException("Resource not found: " + CREDENTIALS_FILE_PATH);
+    }
+    GoogleClientSecrets clientSecrets =
+        GoogleClientSecrets.load(JSON_FACTORY, new InputStreamReader(in));
+
+    // Build flow and trigger user authorization request.
+    GoogleAuthorizationCodeFlow flow =
+        new GoogleAuthorizationCodeFlow.Builder(HTTP_TRANSPORT, JSON_FACTORY, clientSecrets, SCOPES)
+            .setDataStoreFactory(new FileDataStoreFactory(new File(TOKENS_DIRECTORY_PATH)))
+            .setAccessType("offline")
+            .build();
+    LocalServerReceiver receiver = new LocalServerReceiver.Builder().setPort(8888).build();
+    return new AuthorizationCodeInstalledApp(flow, receiver).authorize("me");
+  }
+
+  /**
+   * TODO(): Convert from zonedTimeZone to DateTime
+   *
+   * @param meetingStartTime Time in which the meeting is designated to start at in the format
+   *     "yyyy-mm-dd"
+   * @param timezone String representing the Users Time Zone formatted as an IANA Time Zone Database
+   *     name, e.g. "Europe/Zurich".
+   * @return An EventDateTime with the input meeting time and input time zone.
+   */
+  public EventDateTime setMeetingStartTime(DateTime meetingStartTime, String timezone) {
```

Comment:
Would it be possible to create an event without the timezone string? Currently on the frontend, I am not asking for the user's time zone as the JS Date object seems to capture the timeAvailableUntil and convert it to GMT